### PR TITLE
Create resource iploadbalancing_ssl

### PR DESCRIPTION
**Pull request by beluc**
# Description

Create resource ovh_iploadbalancing_ssl

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] `make testacc TESTARGS="-run TestAccIpLoadbalancingSsl_basic"`
- [x] `make testacc TESTARGS="-run TestAccIpLoadbalancingSsl_importBasic"`

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings or issues
- [x] I have added acceptance tests that prove my fix is effective or that my feature works
- [x] New and existing acceptance tests pass locally with my changes
- [x] I ran succesfully `go mod vendor` if I added or modify `go.mod` file

### CODE REVIEW

File: website/docs/r/iploadbalancing_ssl.html.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Argument Reference
+
+The following arguments are supported:
+
+* `service_name` - (Required) The internal name of your IP load balancing
+* `display_name` - Readable label for loadbalancer ssl
+* `certificate` - Certificate
+* `chain` - Certificate chain
+* `key` - Certificate key
+
```
Comment (amstuta) [inline website/docs/r/iploadbalancing_ssl.html.markdown:35]:
```suggestion
```

File: website/docs/r/iploadbalancing_ssl.html.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+## Attributes Reference
+
+The following attributes are exported:
+
+* `service_name` - See Argument Reference above.
+* `display_name` - See Argument Reference above.
+* `certificate` - See Argument Reference above.
+* `chain` - See Argument Reference above.
+* `key` - See Argument Reference above.
+
```
Comment (amstuta) [inline website/docs/r/iploadbalancing_ssl.html.markdown:46]:
```suggestion
```

File: website/docs/r/iploadbalancing_ssl.html.markdown [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Import
```
Comment (amstuta) [inline website/docs/r/iploadbalancing_ssl.html.markdown:48]: for this to be true, you have to define a func like here: https://github.com/ovh/terraform-provider-ovh/blob/master/ovh/resource_cloud_project_gateway_interface.go#L49

File: ovh/resource_iploadbalancing_ssl_gen.go [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+			Description:         "Fingerprint of your SSL certificate",
+			MarkdownDescription: "Fingerprint of your SSL certificate",
+		},
+		"id": schema.Int64Attribute{
+			CustomType:          ovhtypes.TfInt64Type{},
+			Computed:            true,
+			Description:         "Id of your SSL certificate",
+			MarkdownDescription: "Id of your SSL certificate",
+		},
+		"key": schema.StringAttribute{
```
Comment (amstuta) [inline ovh/resource_iploadbalancing_ssl_gen.go:52]: I think this field should be marked as sensitive

File: ovh/resource_iploadbalancing_ssl_gen.go [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+		"san": schema.ListAttribute{
+			CustomType:          ovhtypes.NewTfListNestedType[ovhtypes.TfStringValue](ctx),
+			Computed:            true,
+			Description:         "Subject Alternative Name of your SSL certificate",
+			MarkdownDescription: "Subject Alternative Name of your SSL certificate",
+		},
+		"serial": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Computed:            true,
+			Description:         "Serial of your SSL certificate (Deprecated, use fingerprint instead !)",
```
Comment (deathiop) [inline ovh/resource_iploadbalancing_ssl_gen.go:68]:
```suggestion
			Description:         "Serial of your SSL certificate (Deprecated, use fingerprint instead!)",
```

File: website/docs/r/iploadbalancing_ssl.html.markdown [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `service_name` - See Argument Reference above.
+* `display_name` - See Argument Reference above.
+* `certificate` - See Argument Reference above.
+* `chain` - See Argument Reference above.
+* `key` - See Argument Reference above.
```
Comment (rbeuque74) [inline website/docs/r/iploadbalancing_ssl.html.markdown:44]: All `Computed` properties should be listed here, `fingerprint`, `expire_date`, `san`, ....

**Review by rbeuque74 — APPROVED** [review]

**Pull request merged**